api/v3/Contribution.php

```diff
@@ -552,9 +552,12 @@ function civicrm_api3_contribution_repeattransaction(&$params) {
   $input = $ids = array();
   civicrm_api3_verify_one_mandatory($params, NULL, array('contribution_recur_id', 'original_contribution_id'));
   if (empty($params['original_contribution_id'])) {
+    //  CRM-19873
+    $is_test = CRM_Core_DAO::getFieldValue('CRM_Contribute_DAO_ContributionRecur', $params['contribution_recur_id'], 'is_test');
     $params['original_contribution_id'] = civicrm_api3('contribution', 'getvalue', array(
       'return' => 'id',
       'contribution_recur_id' => $params['contribution_recur_id'],
+      'contribution_test' => $is_test,
```

Comment:
@h-c-c Can you directly write the call here ? As `$is_test` is not used elsewhere, we can avoid the declaration of it.

Thanks.